### PR TITLE
save the first time each submission is scored

### LINKED ISSUE #1: Record time in database when submission is fully graded

### PR DESCRIPTION
**Pull request by aguss787**
Resolves #1

### CODE REVIEW

File: cms/service/ScoringService.py [hunk 1]
```diff
@@ -30,6 +30,7 @@
 from cms import ServiceCoord, config
 from cms.db import SessionGen, Submission, Dataset, get_submission_results
 from cms.io import Executor, TriggeredService, rpc_method
+from cmscommon import datetime
```
Comment (fushar) [inline cms/service/ScoringService.py:33]: I think we don't need to import this explicitly, because `make_datetime` has already been imported below?

**Review by fushar — APPROVED** [review]

File: cms/service/ScoringService.py [hunk 2]
```diff
@@ -100,6 +100,9 @@ def execute(self, entry):
                 submission_result.ranking_score_details = \
                 score_type.compute_score(submission_result)
 
+            if submission_result.scored_at is None:
+                submission_result.scored_at = make_datetime()
```
Comment (fushar) [inline cms/service/ScoringService.py:104]: Can we add test that checks that it is not None after scoring, somewhere around here: https://github.com/ioi-2022/cms/blob/master/cmstestsuite/unit_tests/service/ScoringServiceTest.py#L100?

**Review by fushar — CHANGES_REQUESTED** [review]

**Review by fushar — APPROVED** [review]
